data/json/recipes/recipe_electronics.json

```diff
@@ -421,6 +434,10 @@
       [ "bottle_plastic", 1 ],
       [ "bottle_plastic_small", 1 ]
     ],
+    [
+      [ "light_bulb", 1 ],
+      [ "lightstrip_dead", 1 ]
```

Comment:
Now that LED light is there, it should be the default result of deconstruction (ie. the first one one the list).